### PR TITLE
PyiModuleGraph: Cleanup initialization.

### PR DESCRIPTION
**Pull request by xoviat**

### CODE REVIEW

File: PyInstaller/depend/analysis.py [hunk 1]
```diff
@@ -544,59 +589,6 @@ def get_co_using_ctypes(self):
         return co_dict
 
 
-# TODO: A little odd. Couldn't we just push this functionality into the
-# PyiModuleGraph.__init__() constructor and then construct PyiModuleGraph
-# objects directly?
```
Comment (xoviat) [inline PyInstaller/depend/analysis.py]: This PR is per this comment.

File: PyInstaller/depend/analysis.py [hunk 2]
```diff
@@ -103,6 +107,47 @@ def __init__(self, pyi_homepath, user_hook_dirs=None, *args, **kwargs):
...
+
+        Returns
+        ----------
+        PyiModuleGraph
+            Module graph with core dependencies.
+        """
+        logger.info('Initializing module dependency graph...')
+
+        if not is_py2:
+            logger.info('Analyzing base_library.zip ...')
```
Comment (htgoebel) [inline PyInstaller/depend/analysis.py]: I'm quite sure, this is done somewhere else again. Please verify and remove any duplicate code.

**Comment by xoviat** [comment]
After looking at this again, I'm not sure this is the best way forward.

**Comment by htgoebel** [comment]
@xoviat May I close this pull-request, or are you still working in it? (IHMO this change is not much important.)

**Pull request closed without merging**